src/main/java/controller/CreateController.java

```diff
@@ -0,0 +1,16 @@
+package controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.swing.tree.ExpandVetoException;
+import java.util.List;
+
+@Controller
+public class CreateController {
+
+    @RequestMapping("/create")
+    public String showCreate(){
```

Comment:
rozumiem, że ma to zwrócić odpowiedni widok, nie można tego jakoś skonfigurować globalnie?